### PR TITLE
Bump server dependencies

### PR DESCRIPTION
**Pull request by scottn12**
This PR bumps server packages to 1036.2000 to 1036.3000. Server package dependencies have been bumped from 1036.2000-0 to 1036.2000. This will be updated to 1036.3000-0 in a future PR once the pre-release is available.

### CODE REVIEW

**Comment by ssimic2** [comment]
I see some bumps to "^0.1036.2001" here.

File: server/routerlicious/lerna.json [hunk 1]
```diff
@@ -2,5 +2,5 @@
   "packages": [
     "packages/**"
   ],
-  "version": "0.1036.2000"
+  "version": "0.1036.2001"
```
Comment (sonalivdeshpande) [inline server/routerlicious/lerna.json:5]: is this expected to change?

File: server/routerlicious/packages/local-server/src/packageVersion.ts [hunk 2]
```diff
@@ -6,4 +6,4 @@
  */
 
 export const pkgName = "@fluidframework/server-local-server";
-export const pkgVersion = "0.1036.2000";
+export const pkgVersion = "0.1036.2001";
```
Comment (sonalivdeshpande) [inline server/routerlicious/packages/local-server/src/packageVersion.ts:9]: here too

File: server/routerlicious/lerna.json (hunk 1 above)
Comment (scottn12) [reply server/routerlicious/lerna.json:5]: I think so, will update these

**Review by ssimic2 — APPROVED** [review]

**Pull request merged**